index.d.ts

```diff
@@ -193,6 +193,42 @@ export interface Options {
 	```
 	*/
 	readonly width?: number;
+
+	/**
+	Set a fixed height for the box.
+
+	__Note__: This option will crop overflowing content.
+
+	@example
+	```
+	import boxen from 'boxen';
+
+	console.log(boxen('foo bar', {height: 5}));
+	// ┌───────┐
+	// │foo bar│
+	// │       │
+	// │       │
+	// └───────┘
+	```
+	*/
+	readonly height?: number;
+
+	/**
+	__boolean__: Wether or not to fit all available space within the terminal.
+
+	__function__: Pass a callback function to control box dimensions.
```

Comment:
This should be clearer about it being used to **modify** the calculated dimensions. And also some use-cases for it.

// @LitoMore